tests/sentry/eventstore/test_models.py

```diff
@@ -105,6 +106,34 @@ def test_email_subject_with_template(self):
 
         assert event1.get_email_subject() == "BAR-1 - production@0 $ baz ${tag:invalid} $invalid"
 
+    def test_transaction_email_subject(self):
+        min_ago = iso_format(before_now(minutes=1))
+        group = self.create_group(type=PerformanceSlowDBQueryGroupType.type_id)
+
+        self.project.update_option(
+            "mail:subject_template",
+            "$shortID - ${tag:environment}@${tag:release} $title",
+        )
+
+        mock_event = Event(
```

Comment:
If you subclass the `PerformanceIssueTestCase` you can do `event = self.create_performance_issue()`

Reply:
Wow that is a _lot_ better!